osm2streets/src/intersection.rs

```diff
@@ -73,6 +73,6 @@ impl StreetNetwork {
 
 impl Intersection {
     pub fn is_border(&self) -> bool {
-        self.control == IntersectionControl::Border
+        self.kind == IntersectionKind::MapEdge
```

Comment:
Thoughts on "border" vs "map edge". Should we rename one to be consistent with the other?

Reply:
`MapEdge` is more clear to me. I chose "border" very long ago